index.ts

```diff
@@ -172,6 +174,52 @@ export class JolocomSDK {
     return token.encode()
   }
 
+  public async rpcDecRequest(req: {
+    toDecrypt: Buffer
+    callbackURL: string
+  }): Promise<string> {
+    const token = await this.idw.create.message(
+      {
+        message: {
+          callbackURL: req.callbackURL,
+          rpc: CallType.AsymDecrypt,
+          request: req.toDecrypt.toString('base64'),
+        },
+        typ: CallType.AsymDecrypt,
+      },
+      await this.bemw.keyChainLib.getPassword(),
+    )
+
+    await this.bemw.interactionManager.start(InteractionChannel.HTTP, token)
+
+    return token.encode()
+  }
+
+  public async rpcEncRequest(req: {
+    toEncrypt: Buffer
+    target: string
+    callbackURL: string
+  }): Promise<string> {
+    const token = await this.idw.create.message(
+      {
+        message: {
+          callbackURL: req.callbackURL,
+          rpc: CallType.AsymEncrypt,
+          request: {
+            data: req.toEncrypt.toString('base64'),
+            target: req.target,
+          },
+        },
+        typ: CallType.AsymEncrypt,
```

Comment:
Do we need to repeat ourselves both in the `rpc` and `typ` values?

Reply:
Also is the `typ` supposed to be the token type (e.g. `generic(?)`)?

Reply:
ah, kind of leftover cruft from the change from explicitly generic tokens to just tokens. can adjust